### PR TITLE
Digital Clock

### PR DESCRIPTION
**Pull request by sahilrawat001**
# Zero To Mastery - Webblocks 2022 📦📦
 Digital Clock made up of html css and vanila javascript .

### CODE REVIEW

File: src/blocks.js [hunk 1]
```diff
@@ -35,6 +35,13 @@
     "author_name": "Dhanush Nehru",
     "author_github_url": "https://github.com/DhanushNehru"
   },
+  {
```
Comment (MattCSmith) [inline src/blocks.js:38]: The block.js file should not be edited directly. Please remove and create a meta.json file (see the readme for more information)

**Review by MattCSmith — CHANGES_REQUESTED** [review]
Looking good so far, however the block.js file should not be edited directly, instead a meta.json file should be created with the relevant information. Please checkout the readme for more information.

**Comment by sahilrawat001** [comment]
Now I've added meta.json

File: src/blocks/SahilRawat/meta.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
[the remark this answers is not in the document]
Comment (sahilrawat001) [reply src/blocks/SahilRawat/meta.json:1]: updated meta.json as per your requirement

**Review by auxfuse — APPROVED** [review]
LGTM 🤜 👓

**Pull request merged**